packages/connect/CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
-| Package                          | stable |    canary    |
-| -------------------------------- | :----: | :----------: |
-| npm @trezor/connect              | 9.2.4  | 9.2.4-beta.2 |
-| npm @trezor/connect-web          | 9.2.4  | 9.2.4-beta.2 |
-| npm @trezor/connect-webextension | 9.2.4  | 9.2.4-beta.2 |
-
-| Connect explorer    | stable |    canary    |
-| ------------------- | :----: | :----------: |
-| connect.trezor.io/9 | 9.2.4  | 9.2.4-beta.2 |
+| Package                          | stable |
+| -------------------------------- | :----: |
+| npm @trezor/connect              | 9.2.4  |
+| npm @trezor/connect-web          | 9.2.4  |
+| npm @trezor/connect-webextension | 9.2.4  |
```

Comment:
```suggestion
| Package                          | stable |    canary    |
| -------------------------------- | :----: | :----------: |
| npm @trezor/connect              | 9.2.4  | - |
| npm @trezor/connect-web          | 9.2.4  | - |
| npm @trezor/connect-webextension | 9.2.4  | - |
```
```suggestion
| Package                          | stable |
| -------------------------------- | :----: |
| npm @trezor/connect              | 9.2.4  |
| npm @trezor/connect-web          | 9.2.4  |
| npm @trezor/connect-webextension | 9.2.4  |
```

Reply:
@Hannsek Which version of the 2 you proposed makes more sense to you?

For me it makes more sense the one with "Canary" column but with `-` because if we do not have canary what is the point of having a column named stable. Any opinions ?

Reply:
Oh sorry, I messed it up somehow. It should be the column with `-`.